gulpfile.js

```diff
@@ -63,38 +63,58 @@ gulp.task( 'exec', ckeditor5DevEnv.execOnRepositories );
 // Compilation tasks. ---------------------------------------------------------
 
 const ckeditor5DevCompiler = require( '@ckeditor/ckeditor5-dev-compiler' );
-const compiler = ckeditor5DevCompiler.compiler( config );
 
 gulp.task( 'default', [ 'compile' ] );
 
-gulp.task( 'compile', callback => {
-	runSequence( 'compile:clean:all', 'compile:themes', 'compile:js', callback );
+gulp.task( 'compile', () => {
+	const args = ckeditor5DevCompiler.utils.parseArguments();
+	const formats = {};
+
+	for ( const item of args.formats ) {
+		formats[ item ] = config.MODULE_DIR[ item ];
+	}
+
+	return ckeditor5DevCompiler.compiler.compile( {
+		formats,
+		packages: ckeditor5DevCompiler.utils.getPackages( config.ROOT_DIR ),
```

Comment:
This should be wrapped into a function, because now the code is repeated 5 times.